File: test/EntityFramework/VBTests/_VbAdvancedPatterns/Partials/AddressMf.vb
Namespace AdvancedPatternsVB

    Partial Public Class AddressMf

        Public Sub New()

        End Sub

        Public Sub New(ByVal street As String, ByVal city As String, ByVal state As String, ByVal zipCode As String, ByVal zone As Integer, ByVal environment As String)
            Me.Street = street
            Me.City = city
            Me.State = state
            Me.ZipCode = zipCode
            SiteInfo = New SiteInfoMf(zone, environment)
        End Sub

    End Class

End Namespace

File: test/EntityFramework/FunctionalTests.Transitional/TestModels/TemplateModels/CsAdvancedPatterns/Partials/AddressMf.cs
// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.

namespace FunctionalTests.ProductivityApi.TemplateModels.CsAdvancedPatterns
{
    public partial class AddressMf
    {
        public AddressMf(string street, string city, string state, string zipCode, int? zone, string environment)
        {
            Street = street;
            City = city;
            State = state;
            ZipCode = zipCode;
            SiteInfo = new SiteInfoMf(zone, environment);
        }
    }
}
